### DISCUSSION TITLE
2022.25.1 Tesla Android hotspot access

### DISCUSSION THREAD

**Discussion by rIcard0-sro** [tesla-android/issue-tracker#53] [Q&A]
hi @mikegapinski 

Congratulations for all your time and your work and especially for this new version!

I don't know if there is a problem but on the point "9: Connect your computer to Tesla Android Wi-Fi network."
We can't connect to the access point because neither the password "raspberry" or "changeit" works , so we can't change it via ssh, and it is no longer possible to do it with a screen and a keyboard. 

Am I the only one in this case? 

Thanks !

**Comment by galosre** [comment]
the problem in the instruction yes you need to connect it to t a screen and keyboard login pi and Pw raspberry
beside got issue with fastboot reboot-fastboot not working use fastboot reboot as the previous one and towards the end got an error

ADB and Fastboot>fastboot format:ext4 userdata
FAILED (remote: failed to set partition)
ADB and Fastboot>fastboot format:ext4 metadata
FAILED (remote: failed to set partition)

**Reply by rIcard0-sro** [reply]
It works perfect, thank you !

**Reply by galosre** [reply]
always on the same tocken once I got the wifi connection still no access ssh in puty ssh pi@3.3.3.1's password:  raspberry is not accepted So something is not enabled!
Using username "ssh pi".
ssh pi@3.3.3.1's password:
Access denied

**Reply by rIcard0-sro** [reply]
Everything worked fine on my side, I had no error with the fastboot reboot-fastboot, the installation is done and the interface is much faster, it's incredible, and I haven't overclocked the pi4 yet and just have a pi2 for capture!

**Comment by galosre** [comment]
always on the same tocken once I got the wifi connection still no access ssh in puty ssh pi@3.3.3.1's password:  raspberry is not accepted So something is not enabled!

**Comment by galosre** [comment]
Ok I did again the android part with another cable this time  USB C to USB C much faster installation specially the platform-tools>fastboot flash super  super.img got it in 6/6sparse before it was 79 /79 and I did not get the errors I mentioned before except Invalid sparse file format at header magic at the different places, but  maybe still a problem since it not very smooth and still do not get the audio definitely no navigation map audio.
Not sure how to use the  Audio Capture app?
